src/smt/optimization_solver.h

```diff
@@ -65,13 +73,15 @@ enum OptResult
 class Objective
 {
  public:
-  Objective(Node n, ObjectiveType type);
+  Objective(Node n, ObjectiveType type, bool bvSignedCompare = true);
```

Comment:
probably the natural default would be `false`?

Reply:
hmmmm... ok~